tools/travis/setup-backend

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 set -e
 set -x
-tools/provision.py --travis
+
+"tools/provision" "--travis"
```

Comment:
Hmm, they were being used a lot in the `install` script, so I assumed they'd be safer. Can remove them.